gallery/src/guidelines/FontSize/FontSizePage.tsx

```diff
@@ -0,0 +1,276 @@
+          and in particular things like accent marks may fall outside of it. Additionally there is no guarantee
+          that glyphs will go right up <em>to</em> the lines. Particularly, there is another number stored in the
+          font, the Capital Height, which indicates the actual (unitless) size of capital letters -
+          capital <q>X</q> to be specific. In OpenSans, this number is 1462, which you might note is smaller than
+          the em-square's Ascent value.
+        </p>
+        <p>
+          Despite the nuances described above, we now get to the first connection with familiar CSS concepts:{' '}
+          <strong>
+            the actual rendered size of the em-square box is what the CSS <code className="nx-code">font-size</code>
+            {' '}property specifies.
+          </strong>
+        </p>
+      </section>
+
+      <section>
+        <h4>Text Node Content Area</h4>
+        <p>
+          As we've established, <code className="nx-code">font-size</code> sets the height of the em-square, a box
+          that is at a lower level than the HTML/CSS model. So what about HTML nodes and elements, what sets their
+          height? Let's start with text nodes, and by extension, simple inline elements that only contain text
+          nodes.
+        </p>
+        <p>
+          In addition to the em-square ascent and descent values described above, fonts also have another set of
+          ascent and descent values which describe line spacing. Actually, it's several sets of values which
+          are each used by different operating systems in different circumstances. Luckily for us however, OpenSans
+          uses a consistent set of values: 2189 for the ascender and 600 for the descender, as pictured below:
+        </p>
+        <img src={openSansLineSpacingImg}/>
+
+        <p>
+          As you can see these values are larger than those that defined the em-square, and they are once again
+          unitless. So what do they do? They help define the line spacing: glyphs outside of the box defined
+          by these values will be cut off, and more notably for web developers, they define the CSS content-height of
+          the text node. So, the box defined by these values is 2189 + 600 = 2789 units tall. This is about 1.36
+          times the height of the em-square, so if we had text with a font-size of 100px, the resulting text node
+          would have a height of about 136px. Notwithstanding any padding or borders, an inline HTML element
+          containing that text node would be that same height, as would the area colored by any background
+          applied to said element. This exact scenario is demonstrated in the live example below:
+        </p>
+
+        <FontLayoutExample/>
+        <CodeExample content={FontLayoutExampleCode}/>
+        <CodeExample language="scss" content={FontLayoutExampleStyles}/>
```

Comment:
I think that it would be worth keeping this "broken" example either here or elsewhere on the page with some simple instructions on how to remediate it via the Web Inspector. It adds a practical example of what can go wrong to the theory explained here.

Reply:
> I think that it would be worth keeping this "broken" example either here or elsewhere on the page with some simple instructions on how to remediate it via the Web Inspector. It adds a practical example of what can go wrong to the theory explained here.

Added another example to deliberately demonstrate this: 84b63ba